app/views.py

```diff
@@ -32,7 +36,29 @@ def exchange_token(backend):
         print(e)
         return jsonify({ 'error': True })
 
-@app.route('/api/poop/', methods=['POST'])
-@api
-def poop():
-    return jsonify({ 'hello': True })
+@app.route('/api/messages', methods=['GET'])
+def get_messages(message_id=None):
```

Comment:
I think this endpoint is great.  We'll have to see how fast/slow it is for different functionality, but for sure we need this endpoint.

That said, I'm not sure this will be good for the github-widget, because getting a years worth of messages sounds like a lot.  It might be faster to do the filtering in SQL (e.g. return only the count of messages, by author, grouped-by date).  Let's merge this and see what works for us.